components/measurement/nettests/WebConnectivity.js

```diff
@@ -332,140 +332,131 @@ const WebConnectivityDetails = ({
   if (isFailure) {
     status = 'error'
     reason = null
-    summaryText = (
-      <FormattedMessage
-        id='Measurement.SummaryText.Websites.Failed'
-        values={{
-          date: date,
-          WebsiteURL: input,
-          network: probe_asn,
-          country: country,
-        }}
-      />
-    )
+    summaryText = <FormattedMessage
+      id='Measurement.SummaryText.Websites.Failed'
+      values={{
+        date,
+        WebsiteURL: <NLink href={getSearchHref(input)}><a>{input}</a></NLink>,
+        network: probe_asn,
+        country
+      }}
+    />
   } else if(isConfirmed) {
     status = 'confirmed'
-    summaryText = intl.formatMessage(
-      {
-        id: 'Measurement.SummaryText.Websites.ConfirmedBlocked'
-      },
-      {
-        date: date,
-        WebsiteURL: input,
+    summaryText = <FormattedMessage
+      id='Measurement.SummaryText.Websites.ConfirmedBlocked'
+      values={{
+        date,
+        WebsiteURL: <NLink href={getSearchHref(input)}><a>{input}</a></NLink>,
         network: probe_asn,
-        country: country,
-      }
-    )
+        country
+      }}
+    />
     headMetadata.message = intl.formatMessage(
       {
         id: 'Measurement.Metadata.WebConnectivity.ConfirmedBlocked',
         defaultMessage: '{hostname} was blocked in {country}'
       },
       {
-        date: date,
+        date,
         hostname,
-        country: country,
+        country,
       }
     )
   } else if (isAnomaly) {
     status = 'anomaly'
     const blockingReason = blocking ?? scores?.analysis?.blocking_type ?? null
     reason = messages[`blockingReason.${blockingReason}`] && intl.formatMessage(messages[`blockingReason.${blockingReason}`])
-    summaryText = (<FormattedMarkdown
+    summaryText = <FormattedMessage
       id='Measurement.SummaryText.Websites.Anomaly'
       values={{
-        date: date,
-        WebsiteURL: input,
+        date,
+        WebsiteURL: <NLink href={getSearchHref(input)}><a>{input}</a></NLink>,
+        'link-to-docs': (string) => (<a href="https://ooni.org/support/faq/#why-do-false-positives-occur">{string}</a>),
```

Comment:
We should also verify if we have access in here to the language of the browser so if we need to localise the URLs we can do that programmatically in here.